classificators/src/main/java/ru/okpdmarket/service/impl/LuceneSearchServiceImpl.java

```diff
@@ -43,6 +43,7 @@
     private static final String CLS_ID = "classificator_code";
     private static final String ITEM_ID = "item_code";
     private static final Logger log = LoggerFactory.getLogger(SearchService.class);
+    private static final Integer NUMBER_OF_HITS = 9999;
```

Comment:
Wouldn't it be better to make it property? And name something like "..search.hits.max"?

Reply:
I considered using a property, but as far as I understand this value won't change so I used constant instead. If I'm wrong, then yeah, gotta change it to prop